chart/templates/_helpers.yaml

```
@@ -607,6 +607,42 @@ Create the name of the cleanup service account to use
   {{- end }}
 {{- end }}
 
+{{define "scheduler_liveness_check_command"}}
+
+  {{- if semverCompare ">=2.0.0" .Values.airflowVersion }}
+  - sh
+  - -c
+  - |
+    CONNECTION_CHECK_MAX_COUNT=0 exec /entrypoint bash -c \
+    "export AIRFLOW__LOGGING__LOGGING_LEVEL=ERROR && export AIRFLOW__CORE__LOGGING_LEVEL=ERROR && \
```

```suggestion
    "export AIRFLOW__LOGGING__LOGGING_LEVEL=ERROR && \
```

With 2.0.0+, this should be enough now.